index.html

```diff
@@ -0,0 +1,856 @@
+            <li><a href="#">Style</a></li>
+            <li><a href="#">T Magazine</a></li>
+            <li><a href="#">Travel</a></li>
+          </ul>
+          <ul>
+            <li class="title">MORE</li>
+            <li><a href="#">Reader Center</a></li>
+            <li><a href="#">Wirecutter</a></li>
+            <li><a href="#">Live Events</a></li>
+            <li><a href="#">The Learning Network</a></li>
+            <li><a href="#">Tools & Services</a></li>
+            <li><a href="#">N.Y.C. Events Guide</a></li>
+            <li><a href="#">Multimedia</a></li>
+            <li><a href="#">Photography</a></li>
+            <li><a href="#">Video</a></li>
+            <li><a href="#">Newsletters</a></li>
+            <li><a href="#">TimesMachine</a></li>
+            <li><a href="#">NYT Store</a></li>
+            <li><a href="#">Times Journeys</a></li>
+            <li><a href="#">Manage My Account</a></li>
+          </ul>
+          <ul id="subscribe">
+            <li class="title">SUBSCRIBE</li>
+            <li>
+              <a href="#"><i class="far fa-newspaper"></i> Home Delivery</a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fab fa-draft2digital"></i> Digital Subscriptions</a
+              >
+            </li>
+            <li>
+              <a href="#"><i class="fas fa-chess-board"></i> Crossword</a>
+            </li>
+            <li>
+              <a href="#"><i class="fas fa-utensils"></i> Cooking</a>
+            </li>
+            <li>
+              <a href="#"><span>Email Newsletters</span></a>
+            </li>
+            <li>
+              <a href="#"><span>Corporate Subscriptions</span></a>
+            </li>
+            <li>
+              <a href="#"><span>Education Rate</span></a>
+            </li>
+
+            <li>
+              <a href="#"><span>Mobile Applications</span></a>
+            </li>
+            <li>
+              <a href="#"><span>Replica Edition</span></a>
+            </li>
+            <li>
+              <a href="#"><span>Español</span></a>
+            </li>
+            <li>
+              <a href="#"><span>中文网</span></a>
+            </li>
+          </ul>
+        </div>
+        <hr />
+        <ul class="footer-end">
+          <li><a>© 2020 The New York Times Company</a></li>
+          <li><a>NYTCo</a></li>
+          <li><a>Contact Us</a></li>
+          <li><a>Work with us</a></li>
+          <li><a>Advertise</a></li>
+          <li><a>T Brand Studio</a></li>
+          <li><a>Your Ad Choices</a></li>
+          <li><a>Privacy</a></li>
+          <li><a>Terms of Service</a></li>
+          <li><a>Terms of Sale</a></li>
+          <li><a>Site Map</a></li>
+          <li><a>Help</a></li>
+          <li><a>Subscriptions</a></li>
+        </ul>
+      </div>
+    </footer>
+
+    <div class="footer-nav">
+      <p>Up-to-date information on coronavirus is available for free</p>
+      <div class="expand">
```

Comment:
- Align `expand` with the other elements. Right now it has extra space on top.
  
  ![Annotation 2020-06-09 224142](https://user-images.githubusercontent.com/57726348/84178749-92a60880-aaa2-11ea-9bb2-fb4fdde157ed.jpg)